### PR TITLE
feat: add home-page files and temp route

### PR DESCRIPTION
**Pull request by BinaHovav**
I laid foundations to start working on the new home-page. 

It's temporarily in a hidden route '/home'.

image

### CODE REVIEW

File: src/img/busImg.png
Comment (NoamGaash) [inline src/img/busImg.png:1]: let's add an attribution section to the `about` page before merging this

Comment (BinaHovav) [reply src/img/busImg.png:1]: okay

File: src/pages/about/index.tsx [hunk 1]
```diff
@@ -138,6 +139,22 @@ const Funding = () => {
...
 
+const Attributions = () => {
+  return (
+    <Widget>
+      <h2>Attributions</h2>
+      Bus ifmage by{' '}
+      <a
+        href="https://www.freepik.com/free-vector/passengers-waiting-bus-city-queue-town-road-flat-vector-illustration-public-transport-urban-lifestyle_10173277.htm#query=public%20transportation&position=0&from_view=search&track=ais&uuid=70a79b38-20cb-42b8-9dde-b96a68088522"
+        target="_blank"
+        rel="noopener noreferrer">
```
Comment (NoamGaash) [inline src/pages/about/index.tsx:150]: shouldn't we use `rel="noopener noreferrer nofollow"`?

**Review by NoamGaash — APPROVED** [review]
Thank you! Having that home page would be great

File: src/pages/about/index.tsx (hunk 1 above)
Comment (BinaHovav) [reply src/pages/about/index.tsx:150]: good idea.

**Comment by NoamGaash** [comment]
great!! :clap:  thank you :)

**Pull request merged**